src/screen/show_refs.rs

```diff
@@ -18,50 +18,40 @@ pub(crate) fn create(config: Rc<Config>, repo: Rc<Repository>, size: Rect) -> Re
         size,
         Box::new(move || {
             let style = &config.style;
-            let head = repo.head().ok();
 
             Ok(iter::once(Item {
-                id: "branches".into(),
+                id: "local_branches".into(),
                 display: Line::styled("Branches".to_string(), &style.section_header),
                 section: true,
                 depth: 0,
                 ..Default::default()
             })
+            .chain(branches(&repo, Some(BranchType::Local), &style.branch)?)
+            .chain(iter::once(Item {
+                id: "remote_branches".into(),
+                display: Line::styled("Remote".to_string(), &style.section_header),
+                section: true,
+                depth: 0,
+                ..Default::default()
+            }))
+            .chain(branches(&repo, Some(BranchType::Remote), &style.branch)?)
```

Comment:
```suggestion
            .chain(branches(&repo, Some(BranchType::Remote), &style.remote)?)
```
There's a style.remote.
In the log remotes / tags / branches are colored after branch/remote/tag configs.
Should this page use the same?

Reply:
Sure, I totally missed it

Reply:
Done